python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/basic_scenarios.py

```diff
@@ -81,7 +81,13 @@
         expected_run_requests=[run_request(asset_keys=["asset1", "asset2"])],
         expected_conditions={
             "asset1": {MissingAutoMaterializeCondition()},
-            "asset2": {MissingAutoMaterializeCondition()},
+            "asset2": {
+                MissingAutoMaterializeCondition(),
+                ParentMaterializedAutoMaterializeCondition(
```

Comment:
we now properly report two conditions at once in these scenarios (both conditions are accurate). this applies to all updated tests in this file.